main.cpp

```diff
@@ -19,20 +19,33 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
-
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {};
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        const auto *cast_expression = Result.Nodes.getNodeAs<CStyleCastexpr>("cast");

```

Comment:
What if `cast_expression` is a null pointer? How to protect the program from crashing in such cases?

Reply:
I can add this:

if (cast_expr == nullptr) {
    return;
}